AnnoDesigner/ViewModels/GeneralSettingsViewModel.cs

```diff
@@ -45,6 +48,19 @@ public GeneralSettingsViewModel(IAppSettings appSettingsToUse)
             SelectedCustomObjectBorderLineColor = savedObjectBorderLineColor.Color;
         }
 
+        private void Commons_SelectedLanguageChanged(object sender, EventArgs e)
+        {
+            var selectedGridLineColorType = SelectedGridLineColor.Type;
+            GridLineColors.Clear();
+            InitGridLineColors();
```

Comment:
Maybe these methods should be renamed to `RefreshGridLIneColors` and `RefreshObjectBorderLineColors` seeing as they are no longer being called only once to initialise the combo boxes when the window is constructed?